staking/message/messages_test.go

```diff
@@ -0,0 +1,207 @@
+package message
+
+// var (
```

Comment:
why are these commented?

Reply:
Because they assumed previous data structure but I want these tests implemented without losing idea of how to be structured, organized. (I prefer to move forward, while giving breadcrumbs for others)